### ISSUE TITLE
echo command missing to display output for signature property - Block signing 

### ISSUE THREAD

**Issue by adiack** [ElementsProject/elementsproject.github.io#42]
Again no big deal but for clarify I would change below.

**********************************
Checking the output of that:

```
COMPLETE=$(echo $BLOCKRESULT | jq '.complete' | tr -d '"')
SIGNBLOCK=$(echo $BLOCKRESULT | jq '.hex' | tr -d '"')
```
We see a result of “True” for the “complete” property as we have signatures from enough keys to satisfy the 2 of 2 requirement. So “complete” in this context means “has enough signatures for the ‘n of m’ multi-sig to be valid”.

Now submit the block, it doesn’t matter who does this as long as they have a signed and valid block hex (which we have stored in the ‘SIGNBLOCK” variable from the results of calling “combineblocksigs”):

`e2-cli submitblock $SIGNBLOCK`
**********************************
Checking the output of that:

```
COMPLETE=$(echo $BLOCKRESULT | jq '.complete' | tr -d '"')
echo $COMPLETE
```
We see a result of “True” for the “complete” property as we have signatures from enough keys to satisfy the 2 of 2 requirement. So “complete” in this context means “has enough signatures for the ‘n of m’ multi-sig to be valid”.

Now **extract and submit** the block, it doesn’t matter who does this as long as they have a signed and valid block hex (which we have stored in the ‘SIGNBLOCK” variable from the results of calling “combineblocksigs”):

```
e2-cli submitblock $SIGNBLOCK
SIGNBLOCK=$(echo $BLOCKRESULT | jq '.hex' | tr -d '"')
```

**Comment by wintercooled** [comment]
Just to clarify - so you mean to emphasise that we pull the signed block hex from the $BLOCKRESULT variable? (extract as you say)

Thinking about it - the following might be better (changed var name to SIGNEDBLOCK and COMBINED as well)...



We now can gather signatures and combine them into a fully signed block:

COMBINED=$(e1-cli combineblocksigs $HEX '''["'''$SIGN1'''", "'''$SIGN2'''"]''')

Checking the output of that:

COMPLETE=$(echo $COMBINED | jq '.complete' | tr -d '"')

We see a result of “True” for the “complete” property as we have signatures from enough keys to satisfy the 2 of 2 requirement. So “complete” in this context means “has enough signatures for the ‘n of m’ multi-sig to be valid”.

Now we will extract the signed block hex from the results of combineblocksig and submit the block. It doesn’t matter who does this as long as they have a signed and valid block hex:

SIGNEDBLOCK=$(echo $COMBINED | jq '.hex' | tr -d '"')
e2-cli submitblock $SIGNEDBLOCK


Is that better do you think @adiack ?

**Comment by adiack** [comment]
Yes much clearer thank you @wintercooled

**Comment by wintercooled** [comment]
I'll do the pull request now then. Thanks again for the feedback.

**Comment by wintercooled** [comment]
I'll do the pull request now then. Thanks again for the feedback @adiack

**Comment by wintercooled** [comment]
Addressed with Pull Request #53
